it/src/main/java/com/google/cloud/teleport/it/jdbc/JDBCResourceManager.java

```diff
@@ -87,7 +89,7 @@ public interface JDBCResourceManager extends ResourceManager {
    *     if the manager object has no dataset, if the table does not exist or if there is an
    *     Exception when attempting to insert the rows.
    */
-  boolean write(String tableName, Map<Integer, List<Object>> rows);
+  boolean write(String tableName, Map<Object, List<Object>> rows);
```

Comment:
Required to make it object? key is the row id which seems to be always int (at least that's what we support today)

Reply:
I changed this to allow non-integer primary keys since I'll be using this for Jdbc to BigQuery load tests.